src/core/jit/model.rs

```diff
@@ -130,11 +133,13 @@ impl<Extensions, Input> Field<Extensions, Input> {
             extensions: self.extensions,
             skip: self.skip,
             include: self.include,
+            pos: self.pos,
             args: self
                 .args
                 .into_iter()
                 .map(|arg| arg.try_map(&map))
                 .collect::<Result<_, _>>()?,
+            is_scalar: self.is_scalar,
```

Comment:
Shouldn't this be inferred from `self.type_of`?

Reply:
This is fine. Ignore my comment.

Reply:
`type_of` is just the name of the type and current checks for scalar requires checking the shape of the type that requires blueprint or similar.